sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Blob/ContainerRegistryBlobClient.cs

```diff
@@ -0,0 +1,503 @@
+        public virtual Response DeleteBlob(string digest, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNullOrEmpty(digest, "digest");
+
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(ContainerRegistryBlobClient)}.{nameof(DeleteBlob)}");
+            scope.Start();
+            try
+            {
+                ResponseWithHeaders<Stream, ContainerRegistryBlobDeleteBlobHeaders> blobResult = _blobRestClient.DeleteBlob(_repositoryName, digest, cancellationToken);
+                return blobResult.GetRawResponse();
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Delete a blob. The blob will only be removed from the repository if no other artifacts reference it.
+        /// </summary>
+        /// <param name="digest">The digest of the blob to delete.</param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <returns></returns>
+        public virtual async Task<Response> DeleteBlobAsync(string digest, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNullOrEmpty(digest, "digest");
+
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(ContainerRegistryBlobClient)}.{nameof(DeleteBlob)}");
+            scope.Start();
+            try
+            {
+                ResponseWithHeaders<Stream, ContainerRegistryBlobDeleteBlobHeaders> blobResult = await _blobRestClient.DeleteBlobAsync(_repositoryName, digest, cancellationToken).ConfigureAwait(false);
+                return blobResult.GetRawResponse();
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Delete a manifest.  Doing so effectively deletes the artifact from the registry.
+        /// </summary>
+        /// <param name="digest">The digest of the manifest to delete.</param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <returns></returns>
+        public virtual Response DeleteManifest(string digest, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNullOrEmpty(digest, "digest");
+
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(ContainerRegistryBlobClient)}.{nameof(DeleteManifest)}");
+            scope.Start();
+            try
+            {
+                return _restClient.DeleteManifest(_repositoryName, digest, cancellationToken);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Delete a manifest.  Doing so effectively deletes the artifact from the registry.
```

Comment:
How are manifests and artifacts related?

Reply:
https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/containerregistry/Azure.Containers.ContainerRegistry#key-concepts

Reply:
would it make sense to say:

```c#
  /// Delete a manifest.  Doing so effectively deletes the artifact described by the manifest from the registry.
```